packages/dd-trace/src/priority_sampler.js

```diff
@@ -29,12 +29,12 @@ const DEFAULT_KEY = 'service:,env:'
 const defaultSampler = new Sampler(AUTO_KEEP)
 
 class PrioritySampler {
-  constructor (env, { sampleRate, rateLimit = 100, rules = [] } = {}) {
+  constructor (env, { sampleRate, rateLimit = 100, samplingRules = [] } = {}) {
```

Comment:
This was supposed to be reverted back to `rules` but it seems it never happened.